docs/UI/SetAppIcon/index.md

```diff
@@ -10,7 +10,14 @@ Purpose
 : Display the application's specified icon in the HMI's application list.
 
 ### Request
+!!! MUST  
 
+1. Keep track and store the the name and file referencing the app icon each time a successful `SetAppIcon` request occurs from the mobile app.  
+2. Display the requested icon together with the application name in the HMI list of registered applications:
+- Right away if the the list of registered applications is currently displayed
```

Comment:
Not formatting correctly, you need to indent new levels of a list with 4 spaces IIRC.

Reply:
@jacobkeeler, do you mean new levels of nested list?
tried to fix https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/83724107cfd1072bb31e6ded9d165593248b05c6